src/sidebar/components/sidebar-content-omega.js

```diff
@@ -0,0 +1,168 @@
+import { createElement } from 'preact';
+import propTypes from 'prop-types';
+import { useEffect, useRef } from 'preact/hooks';
+
+import { withServices } from '../util/service-context';
+import useStore from '../store/use-store';
+import { tabForAnnotation } from '../util/tabs';
+
+import FocusedModeHeader from './focused-mode-header';
+import LoggedOutMessage from './logged-out-message';
+import LoginPromptPanel from './login-prompt-panel';
+import SearchStatusBar from './search-status-bar';
+import SelectionTabs from './selection-tabs';
+import SidebarContentError from './sidebar-content-error';
+import ThreadList from './thread-list';
+
+/**
+ * Render the sidebar and its components
+ */
+function SidebarContent({
+  frameSync,
+  onLogin,
+  onSignUp,
+  loadAnnotationsService,
+  rootThread: rootThreadService,
+  streamer,
+}) {
+  const rootThread = useStore(store =>
+    rootThreadService.thread(store.getState())
+  );
+
+  // store state values
+  const focusedGroupId = useStore(store => store.focusedGroupId());
+  const hasAppliedFilter = useStore(store => store.hasAppliedFilter());
+  const isFocusedMode = useStore(store => store.focusModeEnabled());
+  const isLoading = useStore(store => {
+    return !store.hasFetchedAnnotations() || store.isFetchingAnnotations();
+  });
+  const isLoggedIn = useStore(store => store.isLoggedIn());
+  const linkedAnnotationId = useStore(store =>
+    store.directLinkedAnnotationId()
+  );
+  const linkedAnnotation = useStore(store =>
+    store.findAnnotationByID(linkedAnnotationId)
+  );
+  const searchUris = useStore(store => store.searchUris());
+  const sidebarHasOpened = useStore(store => store.hasSidebarOpened());
+  const userId = useStore(store => store.profile().userid);
+
+  // The local `$tag` of a direct-linked annotation; populated once it
+  // has anchored: meaning that it's ready to be focused and scrolled to
+  const linkedAnnotationAnchorTag =
```

Comment:
Replaces the `ANNOTATIONS_SYNCED` event handling for this component.

Reply:
It looks to me like there are no other consumers of this event in the app, so we can remove that event now.